browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2339,10 +2338,16 @@ L.Control.JSDialogBuilder = L.Control.extend({
 					if (state && state === 'true') {
 						$(button).addClass('selected');
 						$(div).addClass('selected');
+						if (data.command === '.uno:FormatPaintbrush') {
+							$('.leaflet-pane.leaflet-map-pane').addClass('bucket-cursor');
```

Comment:
it would be nice to move this to some global handler, so we will get correct cursor also when no button is created

Reply:
done, thank you for the suggestion